test/performance/upload_data.py

```diff
@@ -107,7 +107,8 @@ def read_metric_data(file_name):
     run_type = content['run_type']
     pull_number = content['pull_number']
     commit_id = content['commit_id']
-
+    execution_timestamp_in_string = execution_timestamp.strftime("%Y-%m-%d %H:%M:%S")
+    print(f"[DEBUG] Type of execution timestamp: {type(execution_timestamp_in_string)}")
```

Comment:
Please remove this debug output. Other than that, LGTM

Reply:
Removed the debug output now.